### ISSUE TITLE
Forbidden character in cookies

### ISSUE THREAD

**Issue by Coder13134** [fryette/webview_cookie_manager#44]
Greetings. I logged in to one of the sites using webviews. Now I am trying to get the cookie, but I get the following error. Apparently, the cookies contain a forbidden character (slash). How can this problem be solved? Before that, everything worked. Appeared with a flutter version update (dart)

![Screenshot_3](https://user-images.githubusercontent.com/86488120/123460033-b7a6e400-d5ef-11eb-8261-66d92d8cdc72.png)
![Screenshot_1](https://user-images.githubusercontent.com/86488120/123460037-b83f7a80-d5ef-11eb-9932-be9883f2a4a4.png)
![Screenshot_2](https://user-images.githubusercontent.com/86488120/123460038-b8d81100-d5ef-11eb-99f0-5a1a3963d801.png)

**Comment by ertgrulll** [comment]
Same here.
![image](https://user-images.githubusercontent.com/65064049/123572708-4e80c580-d7d5-11eb-8edf-14ff657220ce.png)

**Comment by Coder13134** [comment]
> Same here.
> ![image](https://user-images.githubusercontent.com/65064049/123572708-4e80c580-d7d5-11eb-8edf-14ff657220ce.png)

Hi ertgrulll. I solved the given problem as follows:
 - Doing fork this module
 - add next string:
![Screenshot_1](https://user-images.githubusercontent.com/86488120/123597260-c82baa00-d7fb-11eb-9f24-81110e75f12e.png)
- and write in pubspec.yaml:
![image](https://user-images.githubusercontent.com/86488120/123597333-e1ccf180-d7fb-11eb-9faa-738e5511cbc7.png)

**Comment by fryette** [comment]
I would suggest making a PR.
It's really appreciated

**Comment by ertgrulll** [comment]
> > Same here.
> > ![image](https://user-images.githubusercontent.com/65064049/123572708-4e80c580-d7d5-11eb-8edf-14ff657220ce.png)
> 
> Hi ertgrulll. I solved the given problem as follows:
> 
> * Doing fork this module
> * add next string:
>   ![Screenshot_1](https://user-images.githubusercontent.com/86488120/123597260-c82baa00-d7fb-11eb-9f24-81110e75f12e.png)
> * and write in pubspec.yaml (instead original string):
>   ![image](https://user-images.githubusercontent.com/86488120/123597333-e1ccf180-d7fb-11eb-9faa-738e5511cbc7.png)

Hi, thanks for solution. It's working when backslash replaced with "". But it will not work on responses that contains something like "\"", because you are removing all backslashes, removeInvalidCharacter method should change with this: 

```dart
String removeInvalidCharacter(String value) {
    // Remove Invalid Character
    var valueModified = value.replaceAll('\\"', "'").replaceAll("\\", "");
    valueModified = valueModified.replaceAll(String.fromCharCode(32), "");
    return valueModified;
}
```

**Comment by fryette** [comment]
Merged and release the new 2.0.1 version.
Thanks for your contribution @ertgrulll
